### PR TITLE
Fix x64 debug mode build failed on Visual Studio 2015

### PR DESCRIPTION
**Pull request by tamvirtium**
- Fixed compile build on x64 debug for all managed projects.

### CODE REVIEW

File: vtStorManaged/vtStorManaged.vcxproj [hunk 1]
```diff
@@ -43,6 +43,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <PlatformToolset>v120</PlatformToolset>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <UseDebugLibraries>true</UseDebugLibraries>
```
Comment (tamvirtium) [inline vtStorManaged/vtStorManaged.vcxproj:46]: Add missing config to declare debug mode support "UseDebugLibraries"
Default UseDebugLibraries is false that means no using debug libraries

File: vtStorAtaProtocolManaged/vtStorAtaProtocolManaged.vcxproj [hunk 2]
```diff
@@ -43,12 +43,14 @@
     <CharacterSet>Unicode</CharacterSet>
     <CLRSupport>true</CLRSupport>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <UseDebugLibraries>true</UseDebugLibraries>
   </PropertyGroup>
   <PropertyGroup Label="Configuration" Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
     <PlatformToolset>v120</PlatformToolset>
```
Comment (minhvirtium) [inline vtStorAtaProtocolManaged/vtStorAtaProtocolManaged.vcxproj:49]: <PlatformToolset>v120</PlatformToolset> ? v120
we use v140 for VS2015

File: vtStorAtaManaged/vtStorAtaManaged.vcxproj [hunk 3]
```diff
@@ -50,6 +51,7 @@
   <PropertyGroup Label="Configuration" Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
     <PlatformToolset>v120</PlatformToolset>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <UseDebugLibraries>false</UseDebugLibraries>
```
Comment (minhvirtium) [inline vtStorAtaManaged/vtStorAtaManaged.vcxproj:54]: We have some issues with Debug|x64, others mode is ok ? If we change the config, please check

Comment (tamvirtium) [inline vtStorAtaManaged/vtStorAtaManaged.vcxproj:54]: All are done with test

File: vtStorAtaProtocolManaged/vtStorAtaProtocolManaged.vcxproj (hunk 2 above)
Comment (tamvirtium) [inline vtStorAtaProtocolManaged/vtStorAtaProtocolManaged.vcxproj:49]: v140 for VS 2015

**Pull request merged**